torch/nn/attention/_flex_attention.py

```diff
@@ -142,6 +142,71 @@ def _create_block_mask(
     ).as_tuple()
 
 
+def _create_mask(
```

Comment:
I think this is better, for my sanity the actual user facing API will be, 
create_block_mask(score_mod, b, h,m,n, device)? And return our tuple struct?

Reply:
Right! I think ```create_block_mask``` would be the user facing API, we can also consider ```create_mask``` as public as well.